cmd/snap/cmd_download.go

```diff
@@ -129,6 +129,7 @@ func (x *cmdDownload) Execute(args []string) error {
 	if err != nil {
 		return err
 	}
+	fmt.Fprintf(Stdout, i18n.G("Check `snap ack --help` and `snap install --help` for more details.\n"))
```

Comment:
How about providing a more direct hint about what needs to happen for the snap to be installed?

Something along those lines:
```
Install the snap with:
    snap ack actual_filename.assert
    snap install actual_filename.snap
```
It's okay that we're not providing --classic. People need to read the error message when they miss the flag.